alphadia/workflow/base.py

```diff
@@ -82,9 +85,18 @@ def load(
         self.reporter.context.__enter__()
         self.reporter.log_event("section_start", {"name": "Initialize Workflow"})
 
-        self.reporter.log_event("loading_data", {"progress": 0})
         # load the raw data
-        self._dia_data = self._get_dia_data_object(dia_data_path)
+        self.reporter.log_event("loading_data", {"progress": 0})
+        raw_file_manager = RawFileManager(
```

Comment:
this is a very short-lived manager, is this alright?

Reply:
Yes